packages/rainbowkit/src/components/RainbowKitProvider/RainbowKitProvider.tsx

```diff
@@ -50,6 +50,9 @@ export interface RainbowKitProviderProps {
     disclaimer?: DisclaimerComponent;
   };
   coolMode?: boolean;
+  avatar?: {
+    component?: AvatarComponent;
```

Comment:
Could we flatten this to just `avatar`? e.g. `<RainbowKitProvider avatar={CustomAvatar} />`

Reply:
i would do so if we didn't have the `overrideEnsImage` prop, which you mentioned you'd want to remove below